deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -462,3 +516,149 @@ cephObjectStores:
       parameters:
         # note: objectStoreNamespace and objectStoreName are configured by the chart
         region: us-east-1
+
+csi:
+# requests and limits you want to apply for provisioner pod
+  csiRBDProvisionerResource: |
```

Comment:
The csi resources should be defined in the rook-ceph/values.yaml instead of in this rook-ceph-cluster/values.yaml. The other values.yaml already has examples for resources, so we likely just need to uncomment those.